docs/advanced.rst

```diff
@@ -46,7 +63,6 @@ Another example is related to the `feature request #143
 at the beginning and the end of a file.  This use case is easy to implement
 through a custom wrapper for the PEP 8 library::
 
-  #!python
```

Comment:
It didn't seem to be required.

Looking at the [rendered doc](http://pep8.readthedocs.io/en/latest/advanced.html#skip-file-header), it appears that this is not being used to inform syntax highlighting, and is instead being displayed as-is, which is probably not intended.

The other blocks of code appear to be highlighted correctly without this snippet.